pkg/flowaggregator/flowaggregator.go

```diff
@@ -279,6 +279,7 @@ func (fa *flowAggregator) InitCollectingProcess() error {
 			IsEncrypted:   false,
 		}
 	}
+	cpInput.NumExtraElements = len(antreaSourceStatsElementList) + len(antreaDestinationStatsElementList) + len(antreaLabelsElementList)
```

Comment:
it's a bit strange that this doesn't match `aggregationElements` which is what I would expect. I guess I am not familiar enough with go-ipfix.

Reply:
`aggregationElements` contains all elements that needs an update as we receive flow record. The `numExtraElements` are the elements that are added newly after receiving the flow records.